rohmu/object_storage/base.py

```diff
@@ -41,14 +42,15 @@ class Config:
 class BaseTransfer(Generic[StorageModelT]):
     config_model: Type[StorageModelT]
 
-    def __init__(self, prefix, notifier: Optional[Notifier] = None) -> None:
+    def __init__(self, prefix, notifier: Optional[Notifier] = None, statsd_info: Optional[StatsdConfig] = None) -> None:
         self.log = logging.getLogger(self.__class__.__name__)
         if not prefix:
             prefix = ""
         elif prefix[-1] != "/":
             prefix += "/"
         self.prefix = prefix
         self.notifier = notifier or NullNotifier()
+        self.stats = StatsClient(statsd_info)
```

Comment:
if `None` is passed to `parse_obj()`, an error like `expected dict not NoneType (type=type_error)` will be thrown

Reply:
StatsClient checks for statsd_info being `None` before proceding